queries/help/highlights.scm

```diff
@@ -0,0 +1,14 @@
+(tag
+   "*" @_conceal (#set! conceal "")
+   name: (_) @tag)
```

Comment:
I think `@tag` means XML tag for our plugin can you check please?

Reply:
I wouldn't know, that's why I'm asking :)

Reply:
Suggestions very welcome! (as in, just tell me what to use, I have almost no personal opinion on this.)

Reply:
(`@symbol` is no good, though -- it links to `Identifier`, which for programming languages is usually the "default" text color)

Reply:
How about `@keyword`?

Reply:
Yeah, `@label` could work, as these are jump targets, just like `goto` labels.